### PR TITLE
Update icons to version 2.30

### PR DESCRIPTION
**Pull request by mikewheaton**
#### Pull request checklist

- [x] Addresses an existing issue: N/A
- [x] Include a change request file using `$ npm run change`
- [ ] Test the icons

#### Description of changes

Updates to version 2.30 of the icon font.

When reviewing changes to `IconCodes.ts`, see the third commit rather than the overall summary. It's less noisy.

### CODE REVIEW

**Comment by mikewheaton** [comment]
Travis keeps looking for the branch on this repo instead of my fork. I've restarted the build so many times...

File: packages/styling/src/styles/DefaultFontStyles.ts [hunk 1]
```diff
@@ -174,6 +174,7 @@ function _registerDefaultFontFaces(): void {
   if (baseUrl) {
     const fontUrl = `${baseUrl}/fonts`;
     const iconUrl = `${baseUrl}/icons`;
+    const iconVersion = '2.30';
```
Comment (dzearing) [inline packages/styling/src/styles/DefaultFontStyles.ts]: add a change file for styling

Comment (mikewheaton) [inline packages/styling/src/styles/DefaultFontStyles.ts]: Added.

**Comment by mikewheaton** [comment]
@dzearing I'm trying to test this locally to verify that all of the icons display. When I run `gulp serve` from packages/styling I get a blank page because `demo.js` can't be found. Am I missing a step here?

**Pull request merged**